### PR TITLE
Declarative Shadow DOM article

### PR DESCRIPTION
**Pull request by developit**
Changes proposed in this pull request:

- Adds Declarative Shadow DOM article

### CODE REVIEW

File: src/site/content/en/blog/declarative-shadow-dom/index.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+<host-element>
+  #shadow-root (open)
+  <slot>
+    ↳
+    <h2>Light content</h2>
+  </slot>
+</host-element>
+```
+
+This gives us the benefits of Shadow DOM's encapsulation and slot projection in static HTML. No
```
Comment (kaycebasques) [inline src/site/content/en/blog/declarative-shadow-dom/index.md:95]:
```suggestion

{% Aside %}
  The `↳` character represents slotted Light DOM content.
{% endAside %}

This gives us the benefits of Shadow DOM's encapsulation and slot projection in static HTML. No
```

**Review by kaycebasques — APPROVED** [review]
I added an aside to explain that the `↳` character is intentional. I'll merge that suggestion and then check that the new aside renders correctly and then merge this PR.

File: src/site/content/en/blog/declarative-shadow-dom/index.md [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+  #shadow-root (open)
+  <slot>
+    ↳
+    <h2>Light content</h2>
+  </slot>
+</host-element>
+```
+
+{% Aside %}
+  The `↳` character represents slotted Light DOM content.
```
Comment (kaycebasques) [inline src/site/content/en/blog/declarative-shadow-dom/index.md:96]:
```suggestion
  This code sample is following the Chrome DevTools Elements panel's conventions
  for displaying Shadow DOM content. For example, the `↳` character represents slotted Light DOM content.
```

**Review by kaycebasques — APPROVED** [review]
@developit if the aside I added looks OK to you then you can merge this PR whenever you're ready 
image

**Comment by developit** [comment]
Looks good! I'm going to merge tomorrow morning.

**Pull request merged**